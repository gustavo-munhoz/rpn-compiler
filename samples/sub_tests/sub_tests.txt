(10 5 -)
(5 10 -)
(-5 10 -)
(-5 -10 -)
(10 -5 -)
(-10 -5 -)
(1 1 -)
(124 -124 -)
(10 0 -)
(0 10 -)
(-10 0 -)
(0 -10 -)
(0 0 -)
(60000 -6000 -)
(-60000 6000 -)
(0.00006103515625 0.000000059604644775390625 -)
(0.000061094760894775390625 0.00006103515625 -)
(0.000000001 0 -)
(0 0.000000001 -)