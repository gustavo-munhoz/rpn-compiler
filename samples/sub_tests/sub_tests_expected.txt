0x4500
0xC500
0xCB80
0x4500
0x4B80
0xC500
0x0000
0x5BC0
0x4900
0xC900
0xC900
0x4900
0x0000
0x7C00
0xFC00
0x0000
0x0000
0x0000
0x0000